src/config/get-config.ts

```diff
@@ -9,6 +9,10 @@ export interface Config {
    * This is required otherwise the validate-config function will fail.
    */
   TOKEN: string;
+  /*
+   * The connection URI for the MongoDB database.
+   */
+  MONGO_URI: string;
```

Comment:
Add a TODO to make this a validate config required thing. For now it can be optional.

Reply:
Added the note.